.github/workflows/integration-tests-publish.yml

```diff
@@ -16,7 +27,7 @@ jobs:
       id-token: write
       contents: read
     name: Publish Integration Test Image
-    runs-on: ubuntu-latest
+    runs-on: ubuntu20.04-16cores-64GB
```

Comment:
Obligatory: do we need this large of a runner? If so, 👍.

Reply:
It gives us more consistency. The smaller runner was getting close to 9 minute runs with weird occasional 14 to 16 minute runs. We didn't care much before now cause nobody was waiting on this to run. Now we will be using it in place of the integration-tests pipeline to build the test image for manual soak testing so having the more consistent 8 minute run time with the larger runner is more important.